.pfnci/script.sh

```diff
@@ -56,6 +56,12 @@ main() {
           "asia.gcr.io/pfn-public-ci/chainer-ci-prep.${TARGET}" \
           bash /src/.pfnci/run.sh "${TARGET}"
       ;;
+	'chainermn' )
+      docker_args+=(--env="CUPY_VERSION=${CUPY_VERSION:-master}")
+      run "${docker_args[@]}" \
+          "asia.gcr.io/pfn-public-ci/chainermn-ci-prep-${CUDATAG:-cuda92}" \
```

Comment:
What do we do with 'cuda101' version?

Reply:
We can specify the version of CUDA (currently, `cuda92` or `cuda101`) by `CUDATAG`. If not specified, `cuda92` is used as the default value.